src/app/view/components/listing-data-table/listing-data-table.component.ts

```diff
@@ -89,9 +90,19 @@ export class ListingDataTableComponent extends BaseComponent implements OnChange
 
             this.paginatorService
                 .getWithOffset(offset)
-                .subscribe(
-                    items => (this.items = this.dateColumns ? this.formatData(items) : items)
-                );
+                .subscribe(items => this.setItems(items));
+        }
+    }
+
+    /**
+     * Reloads data table with updated data on current page
+     * @memberof ListingDataTableComponent
+     */
+    loadCurrentPage(): void {
```

Comment:
Is this new?

Reply:
Yes, it's new. I use that method to reload the data table when deleting a content type, I was having issues with the currentOffset approach. I call that method here: 
private removeContentType(item): void {
        this.crudService.delete(`v1/contenttype/id`, item.id).subscribe(data => {
            this.listing.loadCurrentPage();
        });
    }

Reply:
Got it!